test/e2e/specs/editor/blocks/list.spec.js

```diff
@@ -1231,4 +1231,25 @@ test.describe( 'List', () => {
 
 		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	test( 'selects all transformed output', async ( { editor, page } ) => {
+		await page.click( 'role=button[name="Add default block"i]' );
+		await page.keyboard.type( '* 1' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2' );
```

Comment:
Since these are just setting up the state for the test, it's not related to the goal of this test. We can use `editor.insertBlock` or `editor.setContent` instead.